docs/release-notes.rst

```diff
@@ -6,6 +6,97 @@
  Release Notes
 ###############
 
+**************
+ Version 0.32
+**************
+
+Version 0.32.0
+==============
+
+**Release Date:** May 03, 2024
+
+Notice: This release includes a fix for the bug posing a risk of data loss. We urge all users on the
+affected versions to upgrade as soon as possible. See ``Bug Fixes`` below for details.
+
```

Comment:
Notice: This release contains an important fix for a bug that poses significant data loss risk. All users on affected versions are strongly encouraged to upgrade as soon as possible. For more details, visit ``Bug Fixes``.

Reply:
my suggested edit doesn't seem to be applied?

Reply:
my fault, forgot to refresh